google-cloud-storage/lib/google/cloud/storage/file/list.rb

```diff
@@ -81,7 +81,7 @@ def next
               prefix: @prefix, delimiter: @delimiter, token: @token, max: @max,
               versions: @versions, user_project: @user_project
             }
-            gapi = @service.list_files @bucket, options
+            gapi = @service.list_files @bucket, **options
```

Comment:
I'd prefer to pass keyword arguments directly here, since there is no `delete_if` on the options hash:

```rb
            gapi = @service.list_files @bucket, prefix: @prefix,
                                                delimiter: @delimiter,
                                                token: @token,
                                                max: @max,
                                                versions: @versions,
                                                user_project: @user_project
```